source/sitemap.xml.erb

```diff
@@ -1,23 +0,0 @@
-<% locale_obj = locals[:locale_obj] || default_locale_obj %>
```

Comment:
Technically this file will 404.  Should we redirect this to `/gatsby/sitemapindex.xml` or something…per locale?  Did Adriana pass this by Jaywing for review?

@Billyhaine can assist with redirects–I expect they will work on a `sitemap.xml`, but I don't believe we have any redirects with "extensions".

Reply:
[Response in this ticket](https://app.asana.com/0/1202529446124674/1202557665702228)